_pages/hotp/2.0.0-crypto-sha256.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Crypto-JS v2.0.0
+ * http://code.google.com/p/crypto-js/
+ * Copyright (c) 2009, Jeff Mott. All rights reserved.
+ * http://code.google.com/p/crypto-js/wiki/License
```

Comment:
From that license page: `Redistributions of source code must retain the above copyright notice, this list of conditions, and the following disclaimer.` Do we need to thus add the text on that website?

Also, would be nice if we didn't have to vendor a minified JS script...

Reply:
It does not mean we need the text on the website